hummingbot/client/ui/style.py

```diff
@@ -109,6 +147,12 @@ def hex_to_ansi(color_hex):
     return "#" + color_ansi
 
 
+text_ui_style = {
+    "&cGREEN": "success-label",
+    "&cYELLOW": "warning-label",
+    "&cRED": "error-label",
```

Comment:
Won't the '&c' be visible when the foreground color is different than the background? 
And if it's made "invisible" wouldn't then the labels be of different lengths?

Reply:
Currently for this PR it is invisible and yes it will be introduce a "padding" before the actual formatted text. I have created another Issue that aims to address this.